### DISCUSSION TITLE
No TeX installation was detected.

### DISCUSSION THREAD

**Discussion by AgedLace** [quarto-dev/quarto-cli#3620] [Q&A]
I'm trying to use GitHub Actions to auto render and publish my blog.
I'm using the GitHub Actions file in the docs located here https://quarto.org/docs/publishing/github-pages.html#publish-action

When I commit to GitHub, I keep getting the "No TeX installation was detected" error.

After researching other discussions regarding the error, I did the following ...

I've run "quarto check" - 
     Checking Quarto installation......OK
      Version: 1.2.269
      Path: /opt/quarto/bin

I've run "quarto tools info tinytex"
{
  "name": "TinyTeX",
  "installed": true,
  "version": "v2022.12",
  "directory": "/home/deb/.TinyTeX",
  "bin-directory": "/home/deb/.TinyTeX/bin/x86_64-linux",
  "configuration": {
    "status": "ok"
  }

I've run "which tlmgr" - had to add the PATH to my .bascrc file
/home/deb/.TinyTeX/bin/x86_64-linux/tlmgr

I've even uninstalled TinyTex and reinstalled it, and I still get this error.

I'm also running the latest quarto in VSCode [latest quarto extension installed] on an updated Manjaro Linux machine.  

Thank you.

**Comment by cscheid** [comment]
> I've even uninstalled TinyTex and reinstalled it, and I still get this error.
> I'm also running the latest quarto in VSCode [latest quarto extension installed] on an updated Manjaro Linux machine.

I think there's some confusion going on. GitHub actions don't run on your computer, they run on GitHub's computers.

Could you share a link to your action and also the log of your github actions so we can track this down?

**Reply by AgedLace** [reply]
GitHub Action file is here:  [publish.yml](https://github.com/AgedLace/Quarto-Blog/tree/main/.github/workflows)

Actions located at [Actions](https://github.com/AgedLace/Quarto-Blog/actions)

Log 
-----
 
...
Quarto Publish
... #10
Jobs
Run details
build-deploy
failed 19 minutes ago in 17s
Search logs
1s
1s
6s
5s
Run quarto-dev/quarto-actions/publish@v2
Run git config --global user.email "quarto-github-actions-publish@example.com"
From https://github.com/AgedLace/Quarto-Blog
 * branch            gh-pages   -> FETCH_HEAD
 * [new branch]      gh-pages   -> origin/gh-pages
Rendering for publish:

[ 1/16] series.qmd
running xelatex - 1

No TeX installation was detected.

Please run 'quarto install tinytex' to install TinyTex.
If you prefer, you may install TexLive or another TeX distribution.

Error: Process completed with exit code 1.
0s
0s

**Comment by cscheid** [comment]
Take a look at the documentation here. You need to configure your action to install tinytex: https://github.com/quarto-dev/quarto-actions/tree/main/setup

**Reply by AgedLace** [reply]
THANK YOU!!!

**Reply by AgedLace** [reply]
Dumb question ... how can I mark this thread as answered?  Your solution worked perfectly, and I appreciate your time and extremely prompt response time!!
